<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Flash_MainWid</class>
 <widget class="QWidget" name="Flash_MainWid">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>578</width>
    <height>478</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MVP3 烧录工具</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="connectionGroup">
     <property name="title">
      <string>串口连接</string>
     </property>
     <layout class="QGridLayout" name="connectionLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="serialPortLabel">
        <property name="text">
         <string>串口:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QComboBox" name="serialPortCombo"/>
      </item>
      <item row="0" column="2">
       <widget class="QLabel" name="baudRateLabel">
        <property name="text">
         <string>波特率:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="3">
       <widget class="QComboBox" name="baudRateCombo"/>
      </item>
      <item row="0" column="4">
       <widget class="QPushButton" name="connectBtn">
        <property name="text">
         <string>连接</string>
        </property>
       </widget>
      </item>
      <item row="0" column="5">
       <widget class="QPushButton" name="disconnectBtn">
        <property name="text">
         <string>断开</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="fileGroup">
     <property name="title">
      <string>烧录文件</string>
     </property>
     <layout class="QGridLayout" name="fileLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="firmwareLabel">
        <property name="text">
         <string>固件文件:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1" colspan="3">
       <widget class="QLineEdit" name="firmwarePathEdit">
        <property name="placeholderText">
         <string>选择固件文件 (.bin)</string>
        </property>
       </widget>
      </item>
      <item row="0" column="4">
       <widget class="QPushButton" name="selectFirmwareBtn">
        <property name="text">
         <string>浏览</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="secureBootLabel">
        <property name="text">
         <string>安全启动:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1" colspan="4">
       <widget class="QPushButton" name="selectSecureBootBtn">
        <property name="text">
         <string>选择安全启动包</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="configLabel">
        <property name="text">
         <string>配置文件:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1" colspan="4">
       <widget class="QPushButton" name="selectConfigBtn">
        <property name="text">
         <string>选择配置文件</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="controlGroup">
     <property name="title">
      <string>烧录控制</string>
     </property>
     <layout class="QVBoxLayout" name="controlLayout">
      <item>
       <layout class="QHBoxLayout" name="progressLayout">
        <item>
         <widget class="QLabel" name="progressLabel">
          <property name="text">
           <string>进度:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QProgressBar" name="progressBar">
          <property name="value">
           <number>0</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="statusLayout">
        <item>
         <widget class="QLabel" name="statusLabel">
          <property name="text">
           <string>状态:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="statusValueLabel">
          <property name="text">
           <string>未连接</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="buttonLayout">
        <item>
         <widget class="QPushButton" name="startBtn">
          <property name="text">
           <string>开始烧录</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="stopBtn">
          <property name="text">
           <string>停止烧录</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="logGroup">
     <property name="title">
      <string>操作日志</string>
     </property>
     <layout class="QVBoxLayout" name="logLayout">
      <item>
       <widget class="QTextEdit" name="logText">
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
